### PR TITLE
CPM-734: Create Page

### PR DESCRIPTION
**Pull request by pierallard**

### CODE REVIEW

**Review by Beninho — APPROVED** [review]

File: components/identifier-generator/front/src/feature/components/CreateGenerator.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import {IdentifierGenerator} from '../../models';
+import {Common} from '../pages/Common';
+
+enum Tabs {
+  GENERAL,
+  PRODUCT_SELECTION,
+  STRUCTURE,
+}
+
+const Container = styled.div`
```
Comment (pierallard) [inline components/identifier-generator/front/src/feature/components/CreateGenerator.tsx:15]: Move to Styled

File: components/identifier-generator/front/src/feature/components/CreateGenerator.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          />
+        </PageHeader.UserActions>
+        <PageHeader.Actions>
+          {/* eslint-disable-next-line @typescript-eslint/no-empty-function */}
+          <Button onClick={/* istanbul ignore next */ () => {} /* TODO */}>;{translate('pim_common.save')}</Button>
+        </PageHeader.Actions>
+        <PageHeader.Title>{translate('pim_title.akeneo_identifier_generator_index')}</PageHeader.Title>
+      </PageHeader>
+      <Container>
+        <TabBar moreButtonTitle="More">
```
Comment (pierallard) [inline components/identifier-generator/front/src/feature/components/CreateGenerator.tsx:52]: translate

File: components/identifier-generator/front/src/feature/components/edit/GeneralProperties.tsx [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+import {LabelTranslations} from './LabelTranslations';
+import {useTranslate} from '@akeneo-pim-community/shared';
+
+type GeneralPropertiesProps = {
+  generator: IdentifierGenerator;
+  onGeneratorChange: (generator: IdentifierGenerator) => void;
+};
+
+const GeneralProperties: React.FC<GeneralPropertiesProps> = ({generator, onGeneratorChange}) => {
+  const defaultIdentifierCode = 'sku'; // TODO
```
Comment (pierallard) [inline components/identifier-generator/front/src/feature/components/edit/GeneralProperties.tsx:15]: TODO

File: components/identifier-generator/front/src/feature/components/edit/GeneralProperties.tsx [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  const translate = useTranslate();
+
+  const onLabelChange = useCallback(
+    (labelCollection: LabelCollection) => {
+      generator.labels = labelCollection;
+      onGeneratorChange({...generator});
```
Comment (pierallard) [inline components/identifier-generator/front/src/feature/components/edit/GeneralProperties.tsx:21]: merge in 1 line

**Review by pierallard — COMMENTED** [review]
mob reviewed

**Review by cemorin — APPROVED** [review]

**Pull request merged**